test/module/irohad/multi_sig_transactions/transport_test.cpp

```diff
@@ -252,4 +250,4 @@ TEST_F(TransportTest, ReplayAttack) {
 
   transport->SendState(&context, &proto_state, &response);
   transport->SendState(&context, &proto_state, &response);
-}
+}
```

Comment:
Please add newline at the end of file. 
To automate, check the link https://stackoverflow.com/a/16761228

Reply:
Thanks for the link! It is fixed now.